nucypher/types.py

```diff
@@ -42,6 +42,7 @@ class WorklockParameters(Tuple):
 
 class StakingEscrowParameters(Tuple):
     seconds_per_period: int
+    days_per_period: int
```

Comment:
I'm not sure why this change is needed?

@vzotova / @cygnusv are there broad implications to adding something here that modifies indices of other other values in the Tuple? Some values from this tuple are retrieved via indices, if I'm not mistaken, so changing the indices could cause issues.

Reply:
Yes, this is a mistake and actually a required change.